include/oneapi/tbb/detail/_task_handle.h

```diff
@@ -136,32 +136,52 @@ class task_dynamic_state {
 };
 #endif // __TBB_PREVIEW_TASK_GROUP_EXTENSIONS
 
+template <typename DerivedType>
+void finalize_base(task_handle_task* p, d1::small_object_allocator& alloc, const d1::execution_data* ed) {
```

Comment:
Can we call the function `destroy_task` or `delete_task`? That's what it does.

Reply:
Done

Reply:
Great idea! Applied